frescobaldi_app/copy2image.py

```diff
@@ -179,7 +170,7 @@ def updateFileTypeUITexts(self):
         Called from translateUI() and from updateExport().
 
         """
-        filetype = self.exportTypes()[self.typeCombo.currentIndex()][1]
+        filetype = _("PNG")
```

Comment:
Nit: Just inline the word “PNG” into the translatable strings?

Reply:
Sure. How do I do that?

Reply:
Ah, that does look a bit cleaner doesn't it.

After pondering the distinction between plain "Copy/Drag" and "Copy/Drag File" I revised the wording further to make it clearer. I'm not sure having them as separate options is actually helpful, to be honest. The plain versions are fine for embedding in a document, while "Save As" offers more control if a file is needed.